cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -78,6 +109,18 @@ func run(*cobra.Command, []string) {
 		return
 	}
 
+	if len(flagBlockHash) == 0 && len(flagStateCommitment) == 0 && len(flagInputPayloadFileName) == 0 {
```

Comment:
I think we should remove this check, and the block hash and state commitment should always be verified. 

Because even though the check was performed when the payload file was generated, the payload itself does not include any information about the previously performed checks, and we are skipping them here.

Reply:
> I think we should remove this check, and the block hash and state commitment should always be verified.

Maybe we should keep this check because it is to ensure that user provides one of these:
- either `flagBlockHash` or `flagStateCommitment` for checkpoint file input
- `flagInputPayloadFileName` for payload file input

If none of these are provided, the program doesn't have input to extract state so it should exit early.